package.json

```diff
@@ -39,7 +39,7 @@
     "launchdarkly-js-client-sdk": "^2.18.1",
     "lodash": "^4.17.20",
     "regenerator-runtime": "^0.13.7",
-    "sbc-common-components": "2.4.19",
+    "sbc-common-components": "2.4.38",
```

Comment:
Another update? 😳 

OK, I've updated bcregistry accordingly.

Reply:
haha yea!
I guess it was about 3hrs ago, and it looks like it was relevant to the account management.